compiler/optimizer/OMRValuePropagation.hpp

```diff
@@ -281,118 +284,32 @@ class ValuePropagation : public TR::Optimization
    ValueConstraint *copyValueConstraints(ValueConstraints &valueConstraints);
 
 #if USE_TREES
-   class ValueConstraintHandler : public TREE_HANDLER<ValueConstraint>
+   class ValueConstraintHandler : public TREE_HANDLER <ValueConstraint>
       {
       public:
-      void setVP(TR::ValuePropagation *vp) {_vp = vp;}
```

Comment:
@andrewcraik Given the concerns from Daryl, plus all other classes are using inheritance to do the mapping, I personally think we should follow the convention we already have, instead of having two ways of doing mapping. What do you think?